src/vs/workbench/contrib/positronOutputWebview/browser/notebookOutputWebviewServiceImpl.ts

```diff
@@ -287,6 +289,11 @@ export class PositronNotebookOutputWebviewService implements IPositronNotebookOu
 
 		// Create the metadata for the webview
 		const webviewInitInfo: WebviewInitInfo = {
+			// TODO: This is what the Viewer pane does. The back layer webview creates a UUID
```

Comment:
I'm not sure about this. I don't actually know why the back layer webview specifies a different `origin` (and therefore a different service worker) for each view type. Maybe it's about security/performance, since service workers are responsible for loading resources?

Currently in this branch, all webview plots will use the same service worker, which is also shared by the viewer's webview.

Reply:
I wonder if this helps deal with the situation where the same asset (e.g. JQuery) may be loaded between two places but are different versions? I know I always run into problems with over-agressive service worker caching. In this case though it seems like the location being different would solve that problem, though.

Reply:
Ah I didn't think of that, makes sense. One big issue is that VSCode doesn't seem to ever clean up a service worker once its created. So I'm tempted to leave this as is and wait until we get user feedback about specific widgets that don't work.